qa/tasks/ceph.py

```diff
@@ -177,12 +177,12 @@ def invoke_logrotate(self):
             while not self.stop_event.is_set():
                 self.stop_event.wait(timeout=30)
                 try:
-                    p = ctx.cluster.run(
-                        args=['sudo', 'logrotate', '/etc/logrotate.d/ceph-test.conf'],
-                        wait=False,
-                        stderr=StringIO()
+                    procs = ctx.cluster.run(
```

Comment:
nit, s/procs/proc/

Reply:
It's actually a list of procs returned by `ctx.cluster.run` so I thought this name was appropriate.